news/2024-05-15-numpy-2-migration.md

```diff
@@ -0,0 +1,9 @@
+# NumPy 2 migration
+
+In preparation for the NumPy 2 release, we have begun a migration of all
+conda-forge packages that depend on `numpy` during the build (so in
+`requirements/host`). The migration status is tracked
+[on our status page](https://conda-forge.org/status/migration/numpy2).
+Particular package's status can be found there. Specific recommendations for
+maintainers updating packages can be found in the migration PRs themselves.
+Please reach out through the usual channels if you have any questions.
```

Comment:
I wouldn't use one of the existing ones (numpy is an order of magnitude larger than CUDA 12.0; around 1000 feedstocks), but if you want to open a new issues that can be spammed with links, fine by me. 👍

Reply:
(FWIW, I mean linking to the issue from the migrator text directly, as was done for CUDA 12)

Reply:
Yeah we could add to the migrator too. Agree there is a tradeoff between visibility in migration PRs and xref overload

From the migrator maybe it would be better to link this news entry instead of an issue. The news entry can then link to the issue. That would reduce xref's , but at the cost of a little more indirection. That might be a good tradeoff though as it would allow more productive discussion in the issue. Plus maintainers could teach each other

No strong feelings though. Just trying to think of different ways we communicate and resolve issues in the migration